Sts1CobcSw/Hal/Spi.hpp

```diff
@@ -10,6 +10,25 @@
 
 namespace sts1cobcsw::hal
 {
+class Spi
+{
+private:
+    RODOS::HAL_SPI spi;
+    RODOS::CommBuffer<std::int64_t> transferEnd;
+
+public:
+    friend auto Initialize(Spi * spiClass, std::int32_t baudRate) -> void;
```

Comment:
Rename `spiClass` to `spi` here and everywhere else that this identifier shows up. There is no value in adding `Class` to the name.